doc/src/vpr/debug_aids.rst

```diff
@@ -24,3 +24,77 @@ Finally, if you need to route an FPGA whose routing architecture cannot be descr
+
+Enabling/Disabling a breakpoint
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enabling and disabling breakpoints are done using the checkboxes in front of each breakpoint in the breakpoint list. The breakpoint is enabled when the box is checked and disabled otherwise.
+
+Deleting a breakpoint
+~~~~~~~~~~~~~~~~~~~~~
+
+Deleting a breakpoint is done using the trash can button in front of each breakpoint in the breakpoint list.
+
+Reaching a breakpoint
+~~~~~~~~~~~~~~~~~~~~~
+
+Upon reaching a breakpoint, the program will stop, notify the user which breakpoint was encountered, and give a summary of the current variable values. This information is presented through a pop-up window and printed to the terminal as well.
+
+Available Variables
+~~~~~~~~~~~~~~~~~~~
+
+.. image:: https://github.com/verilog-to-routing/verilog-to-routing.github.io/blob/master/img/advancedWindow.png
+    :align: center
+
+You can also find the variables’ list in the Advanced Settings Window, on the left.
+
+**Placer Variables**
+
+  * move_num: every placer iteration counts as a move, so the user can stop the program after a certain number of moves. This breakpoint can be enabled through the entry field on the main debugger window or using an expression. It should be noted however, that using the entry field would proceed the specified number of moves. (as in the second example)
+          * Ex. move _num == 33
+          * Ex. move_num += 4
+  * temp_count: every time the temperature is updated it counts as an increase to temp_count. This breakpoint can be enabled through the entry field on the main debugger window or using an expression. It should be noted however, that using the entry field would proceed the specified number of temperatures. (as in the second example)
+          * Ex. temp_count == 5
+          * Ex. temp_count += 5
+  * from_block:  in every placer move one or more blocks are relocated. From_block specifies the first block that is relocated in every move. This breakpoint can be enabled through the entry field on the main debugger window or using an expression.
+          * Ex. from_block == 83
+  * in_blocks_affected: this variable allows you to stop whenever your specified block was moved. Unlike from_block which only checks the first block moved in every move, in_blocks_affected looks through all the replicated blocks in that move. This breakpoint can only be enabled through the use of an expression.
+          * Ex. in_blocks_affected == 1
+    
+**Router Variables**
+
+  * router_iter: every router iteration, whether failed or succeeded, counts as a router iteration. This breakpoint can be enabled through the entry field on the main debugger window or using an expression.
```

Comment:
This is a tautology :).
"Every pass through the whole netlist (with each unrouted or poorly routed net being re-routed) counts as a router iteration."